src/distribution/pert.rs

```diff
@@ -0,0 +1,339 @@
+use distribution;
+use source::Source;
+
+/// A PERT distribution.
+///
+/// The PERT distribution is a family of continuous probability distributions
```

Comment:
I think this long description is not needed. It’s inconsistent with how it’s been so far: none of the implemented distributions is explained. We assume that the reader can look it up if needed.

Reply:
done, though I disagree with the rationale.